package.json

```diff
@@ -144,7 +144,7 @@
     "eslint-plugin-react": "^7.31.8",
     "full-icu": "^1.5.0",
     "glob": "^8.0.3",
-    "graphql-ergonomock": "^1.2.0",
+    "graphql-ergonomock": "npm:@cruglobal/graphql-ergonomock@^1.2.1",
```

Comment:
I figured out that we can use this syntax to use our fork without having to update all of our imports.

Reply:
I didn't know this was a feature. This is a great way to add it!